pkgs/development/python-modules/deezer-python/default.nix

```diff
@@ -0,0 +1,53 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, requests
+, tornado
+, poetry-core
+, pytest
+, pytest-cov
+ ,pytest-vcr
+}:
+
+buildPythonPackage rec {
+  pname = "deezer-python";
+  version = "2.2.2";
+
+  src = fetchFromGitHub {
+    owner = "browniebroke";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1l8l4lxlqsj921gk1mxcilp11jx31addiyd9pk604aldgqma709y";
+  };
+
+  disabled = pythonOlder "3.6";
+
+  format = "pyproject";
+
+  nativeBuildInputs = [
+    poetry-core
+  ];
+
+  checkInputs = [
+    pytest
+    pytest-cov
+    pytest-vcr
+  ];
+
+  checkPhase = ''
+    pytest tests
```

Comment:
Please use pytestCheckHook which makes changing tests in the future easier.

Reply:
Oh, I see. Maybe we also want to change the order [in the docs](https://nixos.org/manual/nixpkgs/stable/#using-pytest) to suggest `pytestCheckHook` first in that case?